llvm/lib/Target/ARM/MCTargetDesc/ARMMCTargetDesc.cpp

```diff
@@ -392,6 +392,18 @@ static MCRelocationInfo *createARMMCRelocationInfo(const Triple &TT,
   return llvm::createMCRelocationInfo(TT, Ctx);
 }
 
+template <typename T, size_t N>
+bool instructionsAreMatching(const T (&Insns)[N], const uint8_t *Buf,
```

Comment:
Can this be made static?

I suggest `instructionsMatch` as a shorter name. Not got a strong opinion.